### ISSUE TITLE
edit-comptext disrupts tab and shift-tab navigation

### ISSUE THREAD

**Issue by catscratchedme** [snowgoon88/TW5-extendedit#31]
When editing a tiddler at http://snowgoon88.github.io/TW5-extendedit/, or editing a tiddler on a new empty.html file with the edit-comptext 0.9.3 plugin installed, the tab and shift-tab navigation is disrupted.

When editing a tiddler, the cursor is focused on the title field by default. In a fresh TW, pressing tab cycles through the following fields in order:
* Tags
* Main text field
* Content type
* Field name
* Field value

When the edit-comptext 0.9.3 plugin is installed, the cursor focuses on an extra step between Tags and Main text field.
![edit-comptext](https://user-images.githubusercontent.com/29561518/171286120-b43029b6-db50-44d1-9431-0ea2e629d591.png)

The main text box is highlighted with a dotted line around its perimeter, but typing does nothing. Pressing tab one more time will bring the cursor back to the Main text field as intended.

**Comment by saqimtiaz** [comment]
I can only recreate this problem in Firefox, and in Firefox this occurs in an empty TiddlyWiki file as well. Therefore it is not a bug related to this plugin.

**Comment by catscratchedme** [comment]
Yes, after testing it again, I can confirm it only happens in Firefox, and happens without your plugin installed. Sorry about getting it wrong!

I'll submit an issue in the main TiddlyWiki github
